### PR TITLE
[QOLSVC-4217] Use builtin jinja truncate() instead of helper's deprecated truncate()

### PR DESCRIPTION
**Pull request by RossWebsterWork**

### CODE REVIEW

**Comment by ThrawnCA** [comment]
The change looks good, but we need to pin the test container to fix Docker Compose.

File: .ahoy.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ commands:
       ahoy title "Building and starting Docker containers"
       sh bin/docker-compose.sh up -d "$@"
       echo "Initialising database schema"
-      ahoy cli '$APP_DIR/bin/init.sh'
+      ahoy cli '$APP_DIR/scripts/init.sh'
```
Comment (ThrawnCA) [inline .ahoy.yml:42]: 'bin' is correct.

Comment (RossWebsterWork) [reply .ahoy.yml:42]: Is bin correct? For ckanext-qa, init.sh currently under /.docker/scripts, not /bin https://github.com/qld-gov-au/ckanext-qa/blob/develop/.docker/scripts/init.sh

Comment (ThrawnCA) [reply .ahoy.yml:42]: Oh, sorry. If the scripts are still in that location then yeah, `scripts` is the way to go.

File: ckanext/qa/templates/report/openness.html [hunk 2]
```diff
@@ -97,7 +97,7 @@
             <td>{{ h.link_to(row['organization_title'], h.url_for('report.org', report_name='openness', organization=row['organization_name'])) }}</td>
           {% endif %}
           <td>{{ row['openness_score'] }}</td>
-          <td class="js-tooltip" title="{{ row['openness_score_reason'] }}">{{ h.truncate(row['openness_score_reason'], 150) }}</td>
+          <td class="js-tooltip" title="{{ row['openness_score_reason'] }}">{{ row['openness_score_reason']|truncate(150) if row['openness_score_reason'] }}</td>
```
Comment (ThrawnCA) [inline ckanext/qa/templates/report/openness.html:100]: Does this null check work correctly if the key doesn't exist on the object at all?

**Review by ThrawnCA — APPROVED** [review]

File: ckanext/qa/templates/report/openness.html (hunk 2 above)
Comment (RossWebsterWork) [reply ckanext/qa/templates/report/openness.html:100]: I don't know if it would, but if it doesn't then I don't think the row title would work either.

**Pull request merged**